app/models/bishop.rb

```diff
@@ -1,4 +1,37 @@
 # frozen_string_literal: true
 
 class Bishop < Piece
+
+	def bishop_valid_move?(x_move, y_move)
+
+		if (x_position - x_move).abs == (y.position - y).abs

```

Comment:
shouldn't line 7 be:
     if (x_position - x_move).abs == (y.position - y_move).abs